src/google.py

```diff
@@ -7,11 +10,52 @@
 from src.constants.http_status_codes import HTTP_200_OK,HTTP_201_CREATED, HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND
 from src.models import UserLogin, UserProfile, db, Borrow
 
+load_dotenv()
+
 
 # Blueprint for google module.
 google_bp = Blueprint('google', __name__, url_prefix='/api')
 
 
+
+def validate_issuer(iss):
+    """Validate the Issuer of the id_token.
+
+    Args:
+        aud (string): Token Issuer
+
+    Returns:
+        bool: True\False. True if the token issuer is correct.
+    """
+    return True if os.environ.get('TOKEN_ISSUER') == iss else False
```

Comment:
I'm not sure why you have this as a function on its own. Also you want to check if os.environ.get('TOKEN_ISSUER') is not empty in the init function where you did checks for other env variables. Doing this comparison removes flexibility of changing things if the values themselves change.